.changeset/healthy-news-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'braid-design-system': patch
+---
+
+DRAFT: Fix tab line position
```

Comment:
We should flesh out some changesets and ensure the correct front matter is there for attribution in the docs site:

This one can be for the tabs active underline change:

```suggestion
---
- Tabs
---

**Tabs**: Improve positioning of the active underline
```


And maybe a separate one for the TooltipRenderer warning fix.